addOns/wappalyzer/src/main/resources/org/zaproxy/zap/extension/wappalyzer/resources/apps.json

```diff
@@ -5728,6 +5728,7 @@
         10
       ],
       "html": [
+        "<script[^<>]*>[^]{0,128}?src\\s*=\\s*['\"]//counter\\.yadro\\.ru/hit(?:;\\S+)?\\?(?:t\\d+\\.\\d+;)?r",
```

Comment:
Left this in to ensure build fails as expected. I should revert before merge. (So that build succeeds).

Reply:
https://travis-ci.com/zaproxy/zap-extensions/jobs/192365059

Reply:
Removed, PoC linked above.